shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -90,6 +90,16 @@ - (instancetype)initWithFrame:(CGRect)frame {
   return self;
 }
 
+// In some scenarios, when we add this view as a maskView of the ChildClippingView, iOS added
+// this view as a
```

Comment:
do you want to cut sentences like this, or does it make sense to manually break the lines below the limit so the autoformater doesn't do something weird?

Reply:
Hmm this is weird. Fixed